test/checks/visibility/hidden-content.js

```diff
@@ -10,37 +10,44 @@ describe('hidden content', function () {
 		}
 	};
 
+	function checkSetup (html, options, target) {
+		fixture.innerHTML = html;
+    axe._tree = axe.utils.getFlattenedTree(fixture);
```

Comment:
Annoying... this file had space/tabs mixed up when I started, and the editor didn't help :/